learning/courses/quantum-machine-learning/classical-ml-review.ipynb

```diff
@@ -0,0 +1,232 @@
+    "\n",
+    "![A diagram of a support vector machine.](/learning/images/courses/quantum-machine-learning/classical-ml-review/svm-cp.avif)\n",
+    "\n",
+    "Let’s suppose a task of binary classification on a dataset with 2-dimensional feature space as shown in the plot. One thing we can do to perform classification for this dataset is to find a line, or in general a hyperplane that separates the two classes. In practice we can find infinitely many separating hyperplanes, so the question is: how do we define the optimal one? The idea here is that a particularly good decision boundary should maximize the margin, which is defined as the distance to the nearest points in each class. In this setting, the data points with the smallest distance to the decision boundary are called support vectors.\n",
+    "\n",
+    "A linear decision boundary could be described in a number of ways; in some ways the most straightforward way is that shown in $f_1$ below. Here, $\\Theta$ is the set of parameters defining the hyperplane, $\\vec{x}$ is your dataset, and $b$ is a constant shift. $\\Phi$ is a mapping from the space of input data points often (but not necessarily) to a higher dimensional space. We will return to this mapping below.\n",
+    "\n",
+    "$$\n",
+    "f_1(\\vec{x}) = \\Theta^T \\Phi(\\vec{x})+b\n",
+    "$$\n",
+    "$$\n",
+    "f_2(\\vec{x}) = \\sum_{i=1}^n \\alpha_i y_i \\Phi^T(\\vec{x}_i)\\Phi(\\vec{x})+b\n",
+    "$$\n",
+    "\n",
+    "In the model $f_1,$ $\\Theta$ is the vector of tunable parameters that the model would learn. This is what we call the “primal formulation”. With some mathematical manipulation we can show that there is a second way we can formulate the same problem, but this time we need to optimize over the alpha parameters shown in the second equation, which we refer to as the “dual formulation”, $f_2$. The main difference is that in the primal formulation the equation has an inner product between the feature vector and the learnable parameters, whereas in the dual formulation the inner product is between feature vectors. Even though the dual form includes both the training data features and the corresponding labels, we will see in the next section how it proves to be more useful than the primal form."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cfba425f-8ebc-4bac-924a-d1f10268df6b",
+   "metadata": {},
+   "source": [
+    "## Kernel methods and how quantum can play a role\n",
+    "\n",
+    "The video below motivates how quantum can play a role in linear classifiers. This is described in greater detail in the text.\n",
+    "\n",
+    "<ImageLink title = \"Quantum kernels\" alt = \"kernel video thumbnail\" href = \"https://video.ibm.com/recorded/134355928\" src = \"/learning/images/courses/quantum-machine-learning/classical-ml-review/cml-review-kernel-thumb2.avif\"/>\n",
```

Comment:
We don't want to use spaces between the argument name and value. it'd be good to search for other places this is happening

```suggestion
    "<ImageLink title=\"Quantum kernels\" alt=\"kernel video thumbnail\" href=\"https://video.ibm.com/recorded/134355928\" src=\"/learning/images/courses/quantum-machine-learning/classical-ml-review/cml-review-kernel-thumb2.avif\"/>\n",
```